### PR TITLE
GLaDOS Copy

### PR DESCRIPTION
**Pull request by MaxWofford**
This includes a bunch of changes to copy to turn Hackbot into GLaDOS.

It's still unfinished: DO NOT MERGE.

### CODE REVIEW

File: lib/data/copy/check_in.yml [hunk 1]
```diff
@@ -5,37 +5,39 @@ first_greeting: >
 
     To start, did you have a club meeting this week?
```
Comment (paked) [inline lib/data/copy/check_in.yml]: Do you want to change the copy on these lines?

File: lib/data/copy/check_in.yml [hunk 2]
```diff
@@ -5,37 +5,39 @@ first_greeting: >
...
 
 meeting_confirmation:
-    positive: Okay, sweet! On which day was it? (say something like "monday" or "today" – I'm not very clever)
-    negative: That's a shame! Was there a particular reason the meeting didn't happen? Is there anything the Hack Club team can be helpful with?
-    invalid: I'm not very smart yet and had trouble understanding you :-/. Try saying something like 'yes' or 'no'.
+    positive: As part of a required test protocol, I am required to tell you that you are doing `quite well`! On which day was it? (say something like "monday" or "today")
+    negative: How unfortunate. Why did the meeting not occur? Is there anything the Hack Club team can be helpful with?
+    invalid: To ensure the safe performance of all Hack Club Weekly Check In activities, please respond with 'yes' or 'no'.
 
 no_meeting_reason: Gotcha. Hope you have a hack-tastic weekend!
```
Comment (paked) [inline lib/data/copy/check_in.yml]: How about:

> I understand. Thank you for participating in the Hack Club Weekly Check In experiment. Goodbye.

File: lib/data/copy/check_in.yml [hunk 3]
```diff
@@ -5,37 +5,39 @@ first_greeting: >
...
 
 day_of_week:
-    unknown: Man, I'm not very smart yet and had trouble understanding you. Try saying something simpler, like "tuesday" or "thursday".
-    invalid: Hmm, that date didn't happen in the past week (although I may be misunderstanding you). Can you try giving me the day of the week of your meeting again?
-    valid: How many people would you estimate came? (I'm not very smart, I'll need you to give me a single number, something like "25" – give your best estimate)
+    unknown: Incomprehensible message. Try saying something simpler, like "tuesday" or "thursday".
```
Comment (paked) [inline lib/data/copy/check_in.yml]: > I am unable to decode this message. Please provide a simpler input, such as "tuesday" or "thursday"

File: lib/data/copy/check_in.yml [hunk 4]
```diff
@@ -5,37 +5,39 @@ first_greeting: >
...
+    invalid: My records indicate this date did not occur in the past week. Can you try giving me the day of the week of your meeting again?
+    valid: How many people would you estimate came? (For example, `25`)
```
Comment (paked) [inline lib/data/copy/check_in.yml]: > Once again, excellent work. Please provide an estimate as to how many attendees your previous meeting captured? (For example, `25`)

File: lib/data/copy/check_in.yml [hunk 5]
```diff
@@ -5,37 +5,39 @@ first_greeting: >
...
+    invalid:
+      - Input invalid. Please input a single number.
+      - Input value invalid. That's the opposite of what input values should be.
+    not_realistic: Make sure your number is a real number greater than 0 (for example, 10 or 25).
+    valid: <%= info[:judgement] %> Is there anything the Hack Club team can be helpful with? I can submit a Hack Club Feedback Report (just make sure to include everything in a single message). If not, please respond with "no".
 
 notes:
-    no_notes: Okay. Hope you have a hack-tastic weekend!
-    had_notes: Sweet, I'll let them know! Hope you have a hactastic weekend!
+    no_notes: Confirmed. Have a good weekend.
```
Comment (paked) [inline lib/data/copy/check_in.yml]: > Your report has been filed. Thank you for helping us help you help us all. Have a good rest break.

File: lib/data/copy/gifs.yml [hunk 6]
```diff
@@ -1,4 +1,31 @@
...
+    invalid:
+      - "Please do not attempt to input invalid commands into Hack Club Hackbot command input. This interface will only respond to the format `@Hackbot gif SEARCH_QUERY`."
+      - "Your input is invalid. That's **literally** the opposite of what your input should be."
+      - 'For your safety, as well as the safety of others, please do not create invalid queries. This interface will only respond to the format `@Hackbot gif SEARCH_QUERY`.'
+      - 'For your safety, as well as the safety of others, please submit valid queries. This interface will only respond to the format `@Hackbot gif SEARCH_QUERY`.'
+      - 'Input invalid: please provide a query.'
+      - 'Input invalid: please tell me something to search for.'
+      - "https://soundcloud.com/user-749646244/no-1"
+      - "https://soundcloud.com/user-749646244/error"
+    valid:
```
Comment (paked) [inline lib/data/copy/gifs.yml]: Could we add:

> Orange is first to acquire a Portal Device. Wait, what?

to this list?

**Comment by paked** [comment]
FYI our convention for branch names is to use `-` as a delimiter.

**Comment by MaxWofford** [comment]
screenshot_2017-03-08_17 17 41

screenshot_2017-03-08_17 17 51

screenshot_2017-03-08_17 17 59

File: lib/data/copy/check_in.yml [hunk 7]
```diff
@@ -3,39 +3,57 @@ first_greeting: >
...
 
-greeting: Hey <%= info[:first_name] %>! Did you have a club meeting this week?
+    Try saying `hackbot gif cave johnson` or `hackbot stats` in a channel that I'm in.
+
+    Let's get started. Did you have a club meeting this week?
+
+greeting: >
+    Hey there!
+
+    Before we get started with check-ins, I just wanted to let you know that I've started cosplaying as GLaDOS and have learned a few new tricks (thanks to some awesome suggestions that y'all sent to my creators).
```
Comment (paked) [inline lib/data/copy/check_in.yml]: Let's provide some additional context as to what GLaDOS is.

File: lib/data/copy/check_in.yml [hunk 8]
```diff
@@ -3,39 +3,57 @@ first_greeting: >
...
+
+    Try saying `hackbot gif cave johnson` or `hackbot stats` in a channel that I'm in.
```
Comment (paked) [inline lib/data/copy/check_in.yml]: We should encourage people to tag hackbot properly:

ie. `@hackbot gif cave johnson`

**Pull request merged**